app-ios/Sources/SearchFeature/ClearBackgroundView.swift

```diff
@@ -0,0 +1,26 @@
+import SwiftUI
+
+struct ClearBackgroundView: UIViewRepresentable {
+    func makeUIView(context: Context) -> some UIView {
+        let view = UIView()
+        DispatchQueue.main.async {
+            view.superview?.superview?.backgroundColor = .clear
+        }
+        return view
+    }
+    func updateUIView(_ uiView: UIViewType, context: Context) {
+    }
+}
```

Comment:
What's that purpose is this view modifier? 👀 
Does not `.background(Color.clear)` work?

Reply:
This modifier erases background color of sheet view. If I replace this modifier with `.background(Color.clear)` The sheet view will be like this.

![Simulator Screen Shot - iPad Air (5th generation) - 2022-10-07 at 15 17 19](https://user-images.githubusercontent.com/22112440/194480948-bb74ed15-c408-4965-a885-56f5074cfcd3.png)

Reply:
Oh, I see.
I think if make background color clear, it can be swiped. Is it correct?
If so, I think this view can be expanded and align content to top.

Reply:
> I think if make background color clear, it can be swiped. Is it correct?
Does "swipe" mean I can swipe with my finger? If you mean that, yes, it still has a hit box.

> If so, I think this view can be expanded and align content to top.
The max height of this view is set to 200px so it can not be expanded. However I don't know why this view is not aligned to top.

Reply:
>  However I don't know why this view is not aligned to top.

Okey. I'll remember about this issue and I'll try to fix this 🙏🏼

Reply:
Thank you! I really appreciate it.

Reply:
Can you revert these changes?

This state may be better. 🙏🏼 
<img width="428" alt="スクリーンショット 2022-10-07 11 19 18" src="https://user-images.githubusercontent.com/30540303/194485493-ad10be0f-988c-4bdb-9c47-b6dc0a6ed018.png">

Reply:
@ry-itto 
I just removed these changes!